src/types/label.js

```diff
@@ -1,34 +1,34 @@
 import {Element} from 'chart.js';
 import {drawBox, drawLabel, measureLabelSize, getChartPoint, toPosition, setBorderStyle, getSize, inBoxRange, isBoundToPoint, resolveBoxProperties, getRelativePosition, translate, rotated, getElementCenterPoint} from '../helpers';
-import {toPadding, toRadians, distanceBetweenPoints, isObject} from 'chart.js/helpers';
+import {toPadding, toRadians, distanceBetweenPoints} from 'chart.js/helpers';
 
 const positions = ['left', 'bottom', 'top', 'right'];
 
 export default class LabelAnnotation extends Element {
 
   inRange(mouseX, mouseY, axis, useFinalPosition) {
-    const {x, y} = rotated({x: mouseX, y: mouseY}, this.getCenterPoint(useFinalPosition), toRadians(-this.options.rotation));
+    const rotation = this.rotation || this.options.rotation;
```

Comment:
why do we need to use both?

Reply:
The label of Line annotation can have rotation equals to auto and Line annotation is the only one which set the rotation as element property,, to store the rotation in order to have the calculated rotation when auto is set

Reply:
The fallback is already invoked at that time I think